### PR TITLE
Dev recski

### PR DESCRIPTION
**Pull request by recski**

### CODE REVIEW

File: brise_plandok/full_attribute_extraction/eval_types_and_values.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                        value_stats["corr"] += 1
+
+            type_stats["preds"].append(p_type)
+            type_stats["golds"].append([g_type])
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description="")
+    parser.add_argument("-f", "--field_to_evaluate")
+    parser.add_argument("-t", "--type", default=False, action="store_true")
```
Comment (Eszti) [inline brise_plandok/full_attribute_extraction/eval_types_and_values.py:39]: Where are you using these two (type and value) fields?

Comment (recski) [reply brise_plandok/full_attribute_extraction/eval_types_and_values.py:39]: https://github.com/recski/brise-plandok/blob/dev_recski/brise_plandok/full_attribute_extraction/eval_types_and_values.py#L52
https://github.com/recski/brise-plandok/blob/dev_recski/brise_plandok/full_attribute_extraction/eval_types_and_values.py#L56

Comment (Eszti) [reply brise_plandok/full_attribute_extraction/eval_types_and_values.py:39]: Ah ok, my bad.

**Review by Eszti — APPROVED** [review]

File: brise_plandok/full_attribute_extraction/predict_modalities.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    "VerbotAufenthaltsraum",
+    "VerbotStellplaetzeUndParkgebaeude",
+}
+
+
+def predict_modality(doc):
+    for sen in doc[DocumentFields.SENS].values():
+        attrs = sen[SenFields.GOLD_ATTRIBUTES] or sen[SenFields.PREDICTED_ATTRIBUTES]
+        if not attrs:
+            sen["predicted_modality"] = None
```
Comment (Eszti) [inline brise_plandok/full_attribute_extraction/predict_modalities.py:35]: - [ ] add predicted_modality to https://github.com/recski/brise-plandok/blob/main/brise_plandok/constants.py#L43

File: brise_plandok/full_attribute_extraction/predict_modalities.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        elif attrs.keys() & PER_ATTRS:
+            sen["predicted_modality"] = "permission"
```
Comment (Eszti) [inline brise_plandok/full_attribute_extraction/predict_modalities.py:37]: - [ ] use the constants: https://github.com/recski/brise-plandok/blob/main/brise_plandok/constants.py#L180

File: brise_plandok/full_attribute_extraction/predict_modalities.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+PER_ATTRS = {
+    "ErrichtungGebaeude",
+    "UnterbrechungGeschlosseneBauweise",
+    "AufbautenZulaessig",
+    "MaxAnzahlGeschosseOberirdischOhneDachgeschoss",
+}
+
+FOR_ATTRS = {
+    "Stockwerk",
```
Comment (Eszti) [inline brise_plandok/full_attribute_extraction/predict_modalities.py:15]: Are you sure about "Stockwerk", "AnOeffentlichenVerkehrsflaechen", "EinfriedungZulaessig", "InSchutzzone" and "HochhausZulaessigGemaessBB"?

File: brise_plandok/full_attribute_extraction/predict_modalities.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    "AnOeffentlichenVerkehrsflaechen",
+    "EinfriedungZulaessig",
+    "VorbautenVerbot",
+    "VerbotStaffelung",
+    "InSchutzzone",
+    "VerbotBueroGeschaeftsgebaeude",
+    "AusnahmeVonWohnungenUnzulaessig",
+    "HochhausZulaessigGemaessBB",
+    "VerbotAufenthaltsraum",
+    "VerbotStellplaetzeUndParkgebaeude",
```
Comment (Eszti) [inline brise_plandok/full_attribute_extraction/predict_modalities.py:27]: - [ ] use the constants https://github.com/recski/brise-plandok/blob/main/brise_plandok/constants.py#L92 (it makes refactoring so much easier if we have to migrate attributes again)

**Comment by Eszti** [comment]
Can you please move the modality scripts to a "modality" subfolder like attribute, value and type?

**Comment by Eszti** [comment]
An EVALUATION.md would also be nice, where we could see the results right away in the repository 😊 But probably, you wanted to do it anyway 😇

**Pull request merged**